stratum/hal/bin/barefoot/deb/start-stratum.sh

```diff
@@ -33,9 +33,10 @@ fi
 PORT_MAP="/etc/stratum/$PLATFORM/port_map.json"
 if [ ! -f "$PORT_MAP" ]; then
     echo "Cannot find port map file $PORT_MAP"
-    exit 255
+    #exit 255
+else
+    ln -f -s "$PORT_MAP" /usr/share/port_map.json
```

Comment:
Can we ship a port_map under the fake `barefoot-tofino-model` platform?

Reply:
I don't think a port map makes much sense for tofino-model. It's supposed to be in lieu of the BSP.